src/doc/src/reference/unstable.md

```diff
@@ -1957,3 +1958,22 @@ default behavior.
 
 See the [build script documentation](build-scripts.md#rustc-check-cfg) for information
 about specifying custom cfgs.
+
+## warnings
+
+The `-Z warnings` feature enables the `build.warnings` configuration option to control how
+Cargo handles warnings. If the `-Z warnings` unstable flag is not enabled, then
+the `build.warnings` config will be ignored.
+
+### `build.warnings`
```

Comment:
Note more for myself.

I easily overlook whether things should be plural or not.  I had to think a second as to whether this should be plural or not.  In this case, its to match the `warnings` group.